src/conn/conn_cache.c

```diff
@@ -8,6 +8,44 @@
 
 #include "wt_internal.h"
 
+/*
+ * __cache_config_abs_to_pct --
+ *	Convert the absolute value in cache config to percentage.
+ */
+static inline int
+__cache_config_abs_to_pct(WT_SESSION_IMPL *session,
+    double *param, const char *param_name, bool shared)
+{
+	WT_CONNECTION_IMPL *conn;
+
+	conn = S2C(session);
+
+	WT_ASSERT(session, param != NULL);
+
+	/*
+	 * Anything above 100 is an absolute value; convert it to percentage.
+	 */
+	if (*param > 100.0) {
+		/*
+		 * In shared cache configuration the cache size changes
+		 * regularly; an absolute value for the configuration parameter
```

Comment:
I don't think a semicolon is necessary here. The two statements are not very tightly tied together. I suggest:
In a shared cache configuration the cache size changes regulary. Therefore, we require a percentage setting and do not allow an absolute size setting.